### DISCUSSION TITLE
Welcome to PSFzf Discussions!

### DISCUSSION THREAD

**Discussion by kelleyma49** [kelleyma49/PSFzf#63] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stinos** [comment]
I'm just here because nothing beats fuzzy matching when looking for something :)

**Comment by kelleyma49** [comment]
Agreed, stinos! To paraphrase the Mandalorian, Fuzzy is the way.
